src/test/java/io/jenkins/plugins/conventionalcommits/JenkinsTest.java

```diff
@@ -162,4 +164,207 @@ public void shouldAddBuildMetadataInformation() throws Exception {
+    // Tag : 0.2.0-alpha / No commit msg
+    URL zipFile = getClass().getResource("simple-project-with-prerelease-tags.zip");
+    assertThat(zipFile, is(notNullValue()));
+
+    p.setDefinition(
+        new CpsFlowDefinition(
+            "node {\n" + "  unzip '" + zipFile.getPath() + "'\n" +
+                "\nnextVersion(preRelease: 'beta')\n" + "}\n",
+            true));
+
+    WorkflowRun b = rule.assertBuildStatus(Result.SUCCESS, p.scheduleBuild2(0).get());
+
+    System.out.println(JenkinsRule.getLog(b));
+
+    assertThat(JenkinsRule.getLog(b), containsString("Started"));
+    assertThat(JenkinsRule.getLog(b), containsString("nextVersion"));
+    assertThat(JenkinsRule.getLog(b), containsString("Current Tag is: 0.2.0-alpha"));
+    assertThat(JenkinsRule.getLog(b), containsString("0.2.1-beta"));
+    assertThat(JenkinsRule.getLog(b), containsString("Finished: SUCCESS"));
+  }
+
+  @Test
+  public void shouldIncrementPreReleaseInformation() throws Exception {
+    WorkflowJob p = rule.jenkins.createProject(WorkflowJob.class, "p");
+    // Tag : 0.2.0-alpha / No commit msg
+    URL zipFile = getClass().getResource("simple-project-with-prerelease-tags.zip");
+    assertThat(zipFile, is(notNullValue()));
+
+    p.setDefinition(
+        new CpsFlowDefinition(
+            "node {\n" + "  unzip '" + zipFile.getPath() + "'\n" +
+                "\nnextVersion(incrementPreRelease: true)\n" + "}\n",
+            true));
+
+    WorkflowRun b = rule.assertBuildStatus(Result.SUCCESS, p.scheduleBuild2(0).get());
+
+    System.out.println(JenkinsRule.getLog(b));
+
+    assertThat(JenkinsRule.getLog(b), containsString("Started"));
+    assertThat(JenkinsRule.getLog(b), containsString("nextVersion"));
+    assertThat(JenkinsRule.getLog(b), containsString("Current Tag is: 0.2.0-alpha"));
+    assertThat(JenkinsRule.getLog(b), containsString("0.2.0-alpha.1"));
+    assertThat(JenkinsRule.getLog(b), containsString("Finished: SUCCESS"));
+  }
+
+  @Test
+  public void shouldBumpCurrentVersionWhenRemovePreRelease() throws Exception {
+    WorkflowJob p = rule.jenkins.createProject(WorkflowJob.class, "p");
+    // Tag : 0.2.0-alpha / No commit msg
+    URL zipFile = getClass().getResource("simple-project-with-prerelease-tags.zip");
+    assertThat(zipFile, is(notNullValue()));
+
+    p.setDefinition(
+        new CpsFlowDefinition(
+            "node {\n" + "  unzip '" + zipFile.getPath() + "'\n" +
+                "\nnextVersion()\n" + "}\n",
+            true));
+
+    WorkflowRun b = rule.assertBuildStatus(Result.SUCCESS, p.scheduleBuild2(0).get());
+
+    System.out.println(JenkinsRule.getLog(b));
+
+    assertThat(JenkinsRule.getLog(b), containsString("Started"));
+    assertThat(JenkinsRule.getLog(b), containsString("nextVersion"));
+    assertThat(JenkinsRule.getLog(b), containsString("Current Tag is: 0.2.0-alpha"));
+    assertThat(JenkinsRule.getLog(b), containsString("0.2.0\n"));
```

Comment:
```suggestion
    assertThat(JenkinsRule.getLog(b), containsString("0.2.0"));
```

A test was failing, when I removed the `\n` it passed :)

Reply:
the test must have the _\n_ to test that it's _0.2.0_ and not _0.2.0-alpha_
And the PR checks (with the test) are ok with this test.